cassandra-release/prepare_release.sh

```diff
@@ -245,7 +245,7 @@ then
     execute "cd cassandra"
     execute "git checkout $release-tentative"
     execute "ant realclean"
-    execute "ant publish -Drelease=true -Dbase.version=$release"
+    execute "ant publish -Drelease=true -Dno-checkstyle=true -Drat.skip=true -Dant.gen-doc.skip=true"
```

Comment:
According to the script name and the action "publish", this is the release preparation step, why don't we need the `gen-doc` step?

We can replace `-Dno-checkstyle=true -Drat.skip=true` with `-Dcheck.skip=true` this will allow us to skip everything above.

Reply:
We don't publish docs with/in releases. ref CASSANDRA-17260